router/boot.go

```diff
@@ -75,9 +84,10 @@ func main() {
 	// FIXME: have to launch cron first so generate-certs will generate the files nginx requires
 	go launchCron()
 
-	waitForInitialConfd(host+":"+etcdPort, timeout)
+	confdHost := strings.Split(etcdPeers[0], "://")[1]
```

Comment:
Here's my `ETCDCTL_PEERS`:
`ETCDCTL_PEERS=http://internal-MT26-05-C-EtcdQuor-16VMRR4RJ8LI8-143835863.us-east-1.elb.amazonaws.com:4001`
I can't explain you why exactly it doesn't work. I've set confd to be more verbose and passed the ETCDCTL_PEERS without the http:// to the `-node` argument, the logs said it was *unable to connect to internal-MT26-05-C-EtcdQuor-16VM*... Then I tried to prefix the argument with `http://` and it worked. As I said, I ended up getting rid of the `.Split`, reinstalled the platform, no more problems.